### PR TITLE
increased default value for MaxSnapshotCountToKeep and MaxChangelogCo…

### PR DESCRIPTION
**Pull request by qurname2**
Increased default values for MaxSnapshotCountToKeep and MaxChangelogCountToKeep

### CODE REVIEW

**Review by achulkov2 — APPROVED** [review]
LGTM.

File: ytop-chart/Chart.yaml [hunk 1]
```diff
@@ -6,3 +6,5 @@ version: 0.0.0-alpha
 appVersion: "0.0.0-alpha"
 sources:
   - https://github.com/ytsaurus/yt-k8s-operator
+# c.HydraManager.MaxSnapshotCountToKeep
+# c.HydraManager.MaxChangelogCountToKeep
```
Comment (l0kix2) [inline ytop-chart/Chart.yaml:10]: what is this?

**Comment by l0kix2** [comment]
need to canonize diff for yson configs.

File: ytop-chart/Chart.yaml (hunk 1 above)
Comment (qurname2) [reply ytop-chart/Chart.yaml:10]: typo

**Pull request merged**